packages/next/src/server/config-shared.ts

```diff
@@ -154,6 +154,11 @@ export interface ExperimentalTurboOptions {
    */
   memoryLimit?: number
 
+  /**
+   * Enable persistent caching for the turbopack dev server and build.
+   */
+  persistentCaching?: boolean
```

Comment:
I think this should be a boolean (use the default values) or an object with config options (e.g. persistence interval duration).